java/src/jmri/jmrit/symbolicprog/EnumVariableValue.java

```diff
@@ -540,14 +540,11 @@ public static class VarComboBox extends JComboBox<String> {
         VarComboBox(ComboBoxModel<String> m, EnumVariableValue var) {
             super(m);
             _var = var;
-            _l = new java.beans.PropertyChangeListener() {
-                @Override
-                public void propertyChange(java.beans.PropertyChangeEvent e) {
-                    if (log.isDebugEnabled()) {
-                        log.debug("VarComboBox saw property change: {}", e);
-                    }
-                    originalPropertyChanged(e);
+            _l = e -> {
+                if (log.isDebugEnabled()) {
+                    log.debug("VarComboBox saw property change: {}", e);
```

Comment:
`if (log.isDebugEnabled()) {` is not needed in this case.

log.isDebugEnabled() is useful if the parameters to log.debug() is costly to calculate.

Reply:
Good point but this was in the existing code.